### PR TITLE
Use namespaced loggers instead of root logger

### LINKED ISSUE #85: Log to 'livereload' logger name instead of root logger
Hi, instead of logging to the rootLogger via 

    import logging
    logging.info('stuff')

I think it would be better to use a named logger via:

    import logging
    logger = logging.getLogger(__name__)
    logger.info('stuff')

This way gives the user more granular control and won't accidentally interfere with the user's settings if they had already set up their root logger.  I'd be happy to provide a PR if you agree.

### PR DESCRIPTION
**Pull request by sloria**
Prevents clashing with user's logging configs

closes #85

### CODE REVIEW

File: livereload/handlers.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 from tornado.web import RequestHandler
 from tornado.util import ObjectDict
 
+logger = logging.getLogger(__name__)
```
Comment (lepture) [inline livereload/handlers.py]: how about `livereload` instead of `__name__`

Comment (sloria) [inline livereload/handlers.py]: Sure. Same with the one in `server.py`?

Comment (lepture) [inline livereload/handlers.py]: @sloria yeah. both are `livereload1

**Comment by sloria** [comment]
@lepture Done. Let me know if there is anything to be done before this is merged.

**Pull request merged**